plugin/trino-kudu/src/main/java/io/trino/plugin/kudu/KuduClientSession.java

```diff
@@ -613,7 +617,16 @@ private KuduSplit toKuduSplit(KuduTableHandle tableHandle, KuduScanToken token,
     {
         try {
             byte[] serializedScanToken = token.serialize();
-            return new KuduSplit(tableHandle.getSchemaTableName(), primaryKeyColumnCount, serializedScanToken, bucketNumber);
+            List<Replica> replicas = token.getTablet().getReplicas();
```

Comment:
Can we hide this feature behind a feature flag ? If the Kudu and Trino are deployed in different clusters then this might not help